src/Bundle/Maker/MakeFactory.php

```diff
@@ -234,14 +264,43 @@ private function defaultPropertiesFor(string $class, bool $allFields): iterable
             $value = "null, // TODO add {$type} {$dbType} type manually";
             $length = $property['length'] ?? '';
 
-            if (\array_key_exists($type, self::DEFAULTS)) {
-                $value = self::DEFAULTS[$type];
+            if (\array_key_exists($type, self::DEFAULTS_FOR_PERSISTED)) {
+                $value = self::DEFAULTS_FOR_PERSISTED[$type];
             }
 
             yield $property['fieldName'] => \str_replace('{length}', (string) $length, $value);
         }
     }
 
+    /**
+     * @param class-string $class
+     */
+    private function defaultPropertiesForNotPersistedObject(string $class, bool $allFields): iterable
```

Comment:
I think in near future this class will be more and more complex, so maybe we should externalise `defaultPropertiesForPersistedObject()` and `defaultPropertiesForNotPersistedObject()` in their own class (maybe 2 class and an interface, I don't know yet).